config.js

```diff
@@ -40,7 +40,8 @@ module.exports = {
     secret: process.env.ICASEWORK_SECRET,
     type: process.env.ICASEWORK_FORM_TYPE || '',
     db: process.env.ICASEWORK_DB,
-    timeout: +process.env.ICASEWORK_TIMEOUT || 60000
+    // timeout: +process.env.ICASEWORK_TIMEOUT || 60000
+    timeout: +process.env.ICASEWORK_TIMEOUT || 2000
```

Comment:
learner question @Alex-Swann what does the plus do here?

Reply:
Short hand way of converting any string number to integer. If it is an integer already it doesn't change it in any way. It's because kube config values always come in as string but if you wanted to run this locally, your command line env vars usually come in the correct type already. So it accommodates both scenarios.

Reply:
ahh beautiful
so this thing https://medium.com/@nikjohn/cast-to-number-in-javascript-using-the-unary-operator-f4ca67c792ce
nice trick, thanks man big props 🙌

Reply:
yeah I saw someone in HMPO stick this in a PR and it made me smile. It's a nice little trick,

Reply:
Technically if you wanted to set ICASEWORK_TIMEOUT to 0 this would go to the default, so maybe
```
+(process.env.ICASEWORK_TIMEOUT || 2000)
```
would be better